### PR TITLE
Fix bug with non-existent organizations

### PR DESCRIPTION
**Pull request by kiritofeng**

### CODE REVIEW

File: judge/views/organization.py [hunk 1]
```diff
@@ -131,8 +132,11 @@ class OrganizationUsers(QueryStringSortMixin, DiggPaginatorMixin, BaseOrganizati
     context_object_name = 'users'
 
     def get_queryset(self):
-        return self.object.members.filter(is_unlisted=False).order_by(self.order).select_related('user') \
-            .defer('about', 'user_script', 'notes')
+        try:
+            return self.object.members.filter(is_unlisted=False).order_by(self.order).select_related('user') \
+                .defer('about', 'user_script', 'notes')
+        except Http404:
+            raise
```
Comment (Xyene) [inline judge/views/organization.py:139]: What is this doing?

Comment (quantum5) [reply judge/views/organization.py:139]: This is literally a no-op, please delete @kiritofeng.

File: judge/views/organization.py [hunk 2]
```diff
@@ -47,7 +48,7 @@ def dispatch(self, request, *args, **kwargs):
         try:
             return super(OrganizationMixin, self).dispatch(request, *args, **kwargs)
         except Http404:
-            key = kwargs.get(self.slug_url_kwarg, None)
+            key = kwargs.get(self.slug_url_kwarg, None) if hasattr(self, 'slug_url_kwarg') else None
```
Comment (quantum5) [inline judge/views/organization.py:51]: I don't know if we want this `hasattr` check. For `AttributeError` to occur, the site would be beyond broken.

**Review by quantum5 — APPROVED** [review]

File: judge/views/organization.py (hunk 2 above)
Comment (kiritofeng) [reply judge/views/organization.py]: Fixed by #2240.

**Pull request merged**